### PR TITLE
Blog post about the new logo

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: content/blog/2023-10-27-new-logo.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+author: Rebecca Sutton Koeser
+thumbnail: /images/posts/dhtech-swag-thumb.png
+featureImage: /images/posts/dhtech-swag.png
+featureImageAlt: photo of a sticker, pen, and pin with variations of new DHtech logo
+featureImageCap: Photo of DHTech swag taken by Rebecca Sutton Koeser
+excerpt: How and why I created the new version of our DHTech logo.
+---
+
+
+Earlier this year, when we were getting ready to order DHTech swag to bring to the DH2024 conference and share with all of you, we hit a snag: we didn't have vector versions of our logo, or even any high-resolution versions.
```
Comment (jdamerow) [inline content/blog/2023-10-27-new-logo.md:13]:
```suggestion
Earlier this year, when we were getting ready to order DHTech swag to bring to the DH2023 conference and share with all of you, we hit a snag: we didn't have vector versions of our logo, or even any high-resolution versions.
```

File: content/blog/2023-10-27-new-logo.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@5/dist/runtime.js";
+import define from "https://api.observablehq.com/d/66f0eb49ce488619@239.js?v=4";
+new Runtime().module(define, Inspector.into("#observablehq-a717a039"));
+</script> -->
+
+I tinkered with d3.js in Observable and came up with something I liked that was fairly close to our old logo, and then I exported that as an SVG and pulled it into Figma. Because we wanted more color, I was experimenting with gradients and colors that would still suggest a metallic shine but not as monotone, and happened on a gradient that reminded me a bit of a CD-ROM. Once I had that idea, I went looking and found a set of [Holographic gradients created by Lily Bather](https://www.figma.com/community/file/1142489146060802380), and chose one that I thought worked well for our logo.
+
+{{< figure src="/logos/dhtech-logo.svg" title="New DHTech logo" class="float_left small">}}
+
+I liked the curve of the letters in the original logo and how they fit inside the gear, and was having trouble finding anything like that, or really any fonts with **d** and **h** that would fit nicely inside the curve of the gear.
```
Comment (jdamerow) [inline content/blog/2023-10-27-new-logo.md:42]:
```suggestion
I liked the curve of the letters in the original logo and how they fit inside the gear, but I was having trouble finding anything like that, or really any fonts with **d** and **h** that would fit nicely inside the curve of the gear. 
```

**Review by jdamerow — APPROVED** [review]
This is great, thank you! Just found one typo and made a minor suggestion :)

File: content/blog/2023-10-27-new-logo.md (hunk 1 above)
Comment (rlskoeser) [reply content/blog/2023-10-27-new-logo.md:13]: 😂 ha, whoops! thanks for catching

**Comment by rlskoeser** [comment]
@jdamerow thanks for the review and edits!

**Pull request merged**